libbeat/scripts/Makefile

```diff
@@ -142,15 +143,18 @@ prepare-tests:
 .PHONY: unit-tests
 unit-tests: ## @testing Runs the unit tests with coverage.  Race is not enabled for unit tests because tests run much slower.
 unit-tests: prepare-tests
+	go test -i ${GOPACKAGES}
```

Comment:
We actually always run `go test -i` followed by `go test`. Just in the case here it is in the line below. Also see line 151 as an example. My tests showed that running `go test -i` followed by `go test` takes the same time together as just running `go test` alone. So why do it? If you run it more then once, `go test -i` is very quick and the run time will be much shorter.